src/open_inwoner/openzaak/api/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path
+
+from open_inwoner.openzaak.api.views import ZakenNotificationsWebhookView
+
+app_name = "openzaak"
+
+urlpatterns = [
+    path(
+        "notifications/webhook/zaken",
+        ZakenNotificationsWebhookView.as_view(),
+        name="notifications-webhook-zaken",
+    ),
```

Comment:
A minor detail..if we want be consistent with the naming conventions of the project we use `notifications_webhook_zaken`

Reply:
Yes we want.